homeassistant/components/plugwise/climate.py

```diff
@@ -0,0 +1,310 @@
+"""
+Plugwise Climate component for HomeAssistant.
+
+Currently supports Anna (not Adam or Lisa yet)
+
+configurations.yaml
+
+climate:
+  - platform: plugwise
+    password: your_short_id   # required, the ID on the smile (some string
+                              # of 6 characters)
+    host: local_ip_address    # required, the IP-address of your smile
+    name: Anna Thermostat     # optional, only if you want to use a
+                              # different name
+    username: smile           # optional, default username is smile
+    port: 80                  # optional, default port is 80
+    scan_interval: 10         # optional, default scan interval is 10 seconds
```

Comment:
These things go into docs pull for. We don't keep such things in code.

Reply:
Removed accordingly